Submitted By: James Cozine
Date: 2004-04-24
Initial Package Version: 3.5.8
Origin: http://lists.freebsd.org/pipermail/freebsd-ports/2003-July/002933.html
Description: Fixes compilation of gv with gcc-3.3.

--- gv-3.5.8.orig/source/paths.h	1997-04-05 15:00:00.000000000 -0700
+++ gv-3.5.8/source/paths.h	2004-04-24 18:41:28.000000000 -0700
@@ -34,9 +34,9 @@
 #   define INC_XMU(aaa) <XMU_DIRECTORY/aaa>
 #   define INC_XAW(aaa) <XAW_DIRECTORY/aaa>
 #else
-#   define INC_X11(aaa) <X11/##aaa##>
-#   define INC_XMU(aaa) <X11/Xmu/##aaa##>
-#   define INC_XAW(aaa) <X11/Xaw3d/##aaa##>
+#   define INC_X11(aaa) <X11/aaa>
+#   define INC_XMU(aaa) <X11/Xmu/aaa>
+#   define INC_XAW(aaa) <X11/Xaw3d/aaa>
 #endif
 
 #endif /* _PATHS_H_ */
